components/compatibility.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Watch for specific plugins and load compatibility if given.
+ */
+function wcsc_plugin_compatibility() {
+	foreach ( wp_get_active_and_valid_plugins() as $item ) {
+		// Turn "woocommerce-german-market/woocommerce-german-market.php" to slug "woocommerce-german-market"
+		$slug = basename( dirname( $item ) );
+
+		// Possible compatibility drop-in
+		$drop_in = __DIR__ . DIRECTORY_SEPARATOR . 'compatibility' . DIRECTORY_SEPARATOR . $slug . '.php';
+
+		if ( ! file_exists( $drop_in ) ) {
```

Comment:
Instead of checking if a file exists, I'd rather use the function `is_plugin_active` to check if the plugin is indeed active: https://codex.wordpress.org/Function_Reference/is_plugin_active

What do you think @ScreamingDev ?

Reply:
`wp_get_active_and_valid_plugins` does that. It is part of the loop.

Reply:
ahh ok, didn't know that 👍